src/main/java/com/epam/ta/reportportal/core/analyzer/impl/AnalyzerServiceImpl.java

```diff
@@ -184,16 +184,18 @@ private void analyzeLaunch(Launch launch, List<TestItem> toAnalyze, IndexLaunch
 			toUpdate.ifPresent(testItem -> {
 				LOGGER.info("Analysis has found a match: {}", analyzed);
 
-				TestItemActivityResource before = TO_ACTIVITY_RESOURCE.apply(testItem, projectId);
-				RelevantItemInfo relevantItemInfo = updateTestItemIssue(projectId, analyzed, testItem);
-				TestItemActivityResource after = TO_ACTIVITY_RESOURCE.apply(testItem, projectId);
-
-				testItemRepository.save(testItem);
-				messageBus.publishActivity(new ItemIssueTypeDefinedEvent(before, after, analyzerInstance, relevantItemInfo));
-				ofNullable(after.getTickets()).ifPresent(it -> messageBus.publishActivity(new LinkTicketEvent(before,
-						after,
-						analyzerInstance
-				)));
+				if (!testItem.getItemResults().getIssue().getIssueType().getLocator().equals(analyzed.getLocator())) {
```

Comment:
could it be null somewhere here ? for example test item without issue?

Reply:
I think no. Items without issue couldn't be analyzed. Only items with issue collects before analysis